distro/otel.go

```diff
@@ -32,7 +32,13 @@ import (
 
 var errShutdown = errors.New("SDK shutdown failure")
 
-const distroVerAttr = "splunk.distro.version"
+// Deprecated
+const splunkVerAttr = "splunk.distro.version"
```

Comment:
How about moving the deprecated constant definitions bellow the "non-deprecated" ones?